recipes/peewee/meta.yaml

```diff
@@ -28,6 +28,13 @@ requirements:
 test:
   imports:
     - playhouse
+    - SqliteDatabase
+    - Model
+    - Query
+    - Field
+    - Proxy
+    - transaction
+    - Database
```

Comment:
My original question was just b/c I was puzzled with the package name vs the import name (`peewee` vs `playhouse`).

Looking at the repo it seems that `playhouse` is the right test though.